core/src/main/java/org/infinispan/transaction/xa/recovery/RecoveryManagerImpl.java

```diff
@@ -123,10 +123,10 @@ public RecoveryIterator getPreparedTransactionsFromCluster() {
    }
 
    @Override
-   public void removeRecoveryInformationFromCluster(Collection<Address> lockOwners, Xid xid, boolean sync, GlobalTransaction gtx) {
+   public void removeRecoveryInformationFromCluster(Collection<Address> lockOwners, Xid xid, boolean sync, GlobalTransaction gtx, boolean skipTxCompletionCommand) {
```

Comment:
Feels like the method name should be `removeRecoveryInformation`, and the parameter name should be `fromCluster`.